docs/_source/tutorials/notebooks/labelling-tokenclassification-basics.ipynb

```diff
@@ -285,13 +289,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# log the dataset in Argilla\n",
-    "rg.log(\n",
-    "    records=ds,\n",
-    "    workspace='recognai',\n",
-    "    name='gutenberg_spacy-ner',\n",
-    "    tags={'with assignments': True}\n",
-    ")"
+    "# loop through the assignments dictionary to save the metadata and log the records in Argilla\n",
+    "for user, records in assignments.items():\n",
```

Comment:
Hi Natalia, I think it makes more sense to first create a list of the total record and afterwards log all of them at once. Thank you:)

```python
assigned_records = []
for user, records in assignments.items():
    for r in records:
        r.metadata['user'] = user
        assigned_records.append(r)
rg.log(
    records=records,
    workspace='recognai',
    name='gutenberg_spacy-ner',
    tags={'with assignments': True}
)
```